src/parser.js

```diff
@@ -259,6 +268,11 @@ class Parser {
 
         const section = this.getSection(line);
 
+        if (this.isInheritedSection(line)) {
+            const parentSection = this.getParentSection(line)
+            ctx.ini[section] = JSON.parse(JSON.stringify(ctx.ini[parentSection]))
```

Comment:
Is there any specific reason to do the parse/stringify?

Reply:
Hi - i used JSON transforming, because spread syntax as well as Object.assign() does not allow deep cloning, so this is a workaround for deeply nested ini values. I also added test for this case in the updated branch.

Reply:
Thx for clarifying 😄